modules/account-lib/src/coin/cspr/transaction.ts

```diff
@@ -209,18 +213,15 @@ export class Transaction extends BaseTransaction {
       const argValue = 1;
 
       const transferValues = new Map();
-      transferValues.set(
-        TRANSACTION_TYPE,
-        (this.casperTx.session.getArgByName(TRANSACTION_TYPE) as CLValue).asString(),
-      );
+      transferValues.set(TRANSACTION_TYPE, (this.casperTx.session.getArgByName(TRANSACTION_TYPE) as CLString).value());
       transferValues.set('amount', getTransferAmount(this.casperTx.session));
       transferValues.set('to_address', getTransferDestinationAddress(this.casperTx.session));
       const transferId = getTransferId(this.casperTx.session);
       if (transferId !== undefined) {
         transferValues.set('id', transferId.toString());
       }
 
-      txJson['deploy']!['session']['Transfer']['args'].forEach((arg) => {
```

Comment:
what's the reason to remove the non-null assertion operator?

Reply:
Our TS config gives a warning, i can leave them but pretty much anything in that JSON is an any since the deployToJson method from CSPR doesnt have defined type.